src/Service/FourLetterCommand.cpp

```diff
@@ -45,9 +45,10 @@ String IFourLetterCommand::toName(int32_t code)
 
 int32_t IFourLetterCommand::toCode(const String & name)
 {
-    int32_t res = *reinterpret_cast<const int32_t *>(name.data());
+    int32_t res;
+    std::memcpy(&res, name.data(), sizeof(res));
     /// keep consistent with Coordination::read method by changing big endian to little endian.
-    return __builtin_bswap32(res);
```

Comment:
why not use` __builtin_bswap32`

Reply:
std::byteswap is standardized in C++23, it better than use __builtin_bswap32 which is a GCC/Clang built-in function.

Reply:
Please find and replace `__builtin_bswap32 ` in global

Reply:
okay